### PR TITLE
docs: Add how to use iotivity

### PR DESCRIPTION
**Pull request by heejin-kim**
To use IoTivity feature, there are IoTivity information and build guide.

### CODE REVIEW

File: docs/HowToUseIotivity.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+How to use IoTivity
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: Let's use '#' instaed of "====".

File: docs/HowToUseIotivity.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+===================
+
+IoTivity is an open source software framework enabling seamless device-to-device connectivity to address the emerging needs of the Internet of Things.
+If you want to know more detailed information, please find it at [README](../external/iotivity/iotivity_1.2-rel/README.md) and https://iotivity.org
+
+----------
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: Let's remove.

File: docs/HowToUseIotivity.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+How to use IoTivity
+===================
+
+IoTivity is an open source software framework enabling seamless device-to-device connectivity to address the emerging needs of the Internet of Things.
+If you want to know more detailed information, please find it at [README](../external/iotivity/iotivity_1.2-rel/README.md) and https://iotivity.org
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: 1.  how about below?
```
Please find details at README (link) and Iotivity site (link).
```
I heard we had better not use "you" on this kind of doc.
2. If you want to split a line from above, you shoudl add two spaces at end of above line.

File: docs/HowToUseIotivity.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+----------
+##To use the IoTivity feature
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: 1. Add a space
2. Here is only how to use pre-set config. When someone wants to enable iotivity from own config, he can't know how to do it.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToUseIotivity.md (hunk 1 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: I see

File: docs/HowToUseIotivity.md (hunk 3 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: Ok

File: docs/HowToUseIotivity.md (hunk 2 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: Ok

File: docs/HowToUseIotivity.md (hunk 4 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: I see

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToUseIotivity.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# How to use IoTivity
+
+IoTivity is an open source software framework enabling seamless device-to-device connectivity to address the emerging needs of the Internet of Things.  
+More detailed information, please find it at [README](../external/iotivity/iotivity_1.2-rel/README.md) and [IoTivity Site](https://iotivity.org)
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: Please add '.' at end of line.

File: docs/HowToUseIotivity.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# How to use IoTivity
+
+IoTivity is an open source software framework enabling seamless device-to-device connectivity to address the emerging needs of the Internet of Things.  
+More detailed information, please find it at [README](../external/iotivity/iotivity_1.2-rel/README.md) and [IoTivity Site](https://iotivity.org)  
+
+## To use the IoTivity feature
+
+### 1. with a build configuration of IoTivity
+
+Please find it at [README](../external/iotivity/README.md)
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: Same as previous

File: docs/HowToUseIotivity.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+### 2. with another build configuration
+
+Set the Iotivity feature through menuconfig.  
+```
+~/$ cd $TIZENRT_BASEDIR
+~/TIZENRT_BASEDIR$ cd os
+~/TIZENRT_BASEDIR/os$ make menuconfig
+```
+In menuconfig, select the 'enable / disable iotivity stack'
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: same

File: docs/HowToUseIotivity.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+In menuconfig, select the 'enable / disable iotivity stack'  
+```
+    -+-External Libraries
+          |
+          +-enable / disable iotivity stack
+```
+
+## To build IoTivity sample app - simple server
+
+Please find it at [README](../apps/examples/iotivity_simpleserver/README.md)
```
Comment (sunghan-chang) [inline docs/HowToUseIotivity.md]: same

File: docs/HowToUseIotivity.md (hunk 5 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: modified

File: docs/HowToUseIotivity.md (hunk 6 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: modified

File: docs/HowToUseIotivity.md (hunk 7 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: modified

File: docs/HowToUseIotivity.md (hunk 8 above)
Comment (heejin-kim) [reply docs/HowToUseIotivity.md]: modified

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**